src/components/CurrencyToggler/CurrencyToggler.js

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import {
+  DropdownItemContainer,
+  SelectedWrapper,
+  Img,
+  DownArrow,
+  DropdownWrapper,
+  CurrencyWrapper,
+} from "./CurrencyToggler.styles";
+
+function DropdownItem(props) {
+  return (
+    <DropdownItemContainer onClick={() => props.handleClick(props.currency)}>
+      <Img src={props.symbol} alt={`${props.curency} symbol`} />
+      <div>{props.currency}</div>
+    </DropdownItemContainer>
+  );
+}
+
+class CurrencyToggler extends React.Component {
+  state = {
+    isExpanded: false,
+    currencies: [
+      {
+        name: "USD",
+        isActive: true,
+        symbolURL: "https://i.ibb.co/YkKkc6J/dollar-icon.png",
+      },
+      {
+        name: "GBP",
+        isActive: false,
+        symbolURL: "https://i.ibb.co/cNSyGZP/pound-icon.png",
+      },
+      {
+        name: "EUR",
+        isActive: false,
+        symbolURL: "https://i.ibb.co/tP0n42j/euro-icon.png",
+      },
+    ],
+  };
+
+  handleClick = () => {
+    this.setState({ isExpanded: !this.state.isExpanded });
+  };
+
+  handleSelect = (selectedCurrency) => {
+    this.state.currencies.map((currency) => {
+      return currency.name === selectedCurrency
+        ? (currency.isActive = true)
+        : (currency.isActive = false);
+    });
```

Comment:
this.state.currencies.map((currency) => {
currency.isActive =  currency.name === selectedCurrency;
return currency
   
    });

Reply:
This one is brilliant. love it.
When returning from mapping, is it a convention to return the element of the array or can I just use the return keyword alone?